packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -264,7 +267,7 @@ export class InternalDropdownItem extends React.Component<InternalDropdownItemPr
               role={role}
               onKeyDown={this.onKeyDown}
               onClick={(event: any) => {
-                if (!isDisabled) {
+                if (!isDisabled && !isAriaDisabled) {
```

Comment:
Should this be an `||` instead? Or does `isDisabled` also set `isAriaDisabled` if `isAriaDisabled` isn't passed in.

Reply:
Good question.  Maybe add a test for this scenario.

Reply:
I may be going cross eyed - but i think it's true that if the dropdown item is disabled OR ariaDisabled, then we don't want to fire the passed in onClick or onSelect handlers. So this logic, with the `&&` is right.

Reply:
Oh you're right, the double negative with the && got me

Reply:
Can rewrite as `isDisabled || isAriaDisabled` to be more clear. de Morgan's theorem states `!A && !B = A || B`.